.travis.yml

```diff
@@ -1,15 +1,15 @@
-# http://docs.travis-ci.com/user/workers/container-based-infrastructure/
+# https://docs.travis-ci.com/user/reference/overview/
 language: php
-sudo: false
 
-# Travis support wrote "Ubuntu 14.04 reached EOL on April 30th, 2019,
-# ... we've been slowly rolling out changes to make Xenial builds ..."
-# https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment
-# https://docs.travis-ci.com/user/reference/xenial/#services-disabled-by-default.
+os:
+  - linux
+
+dist: xenial 
```

Comment:
I'd not add the dist as this causes need for update every now and then. I've never seen a warning caused by this key not being there. Same for the os key.

Reply:
> I'd not add the dist as this causes need for update every now and then.

You are right here. We could leave this to travis to decide and use newer versions as soon as they default to is.

> I've never seen a warning caused by this key not being there. 

Dunno when this started. I just accidentally stumbled on it a couple of days before.

> Same for the os key.

Well since we are testing on Linux anyways we could tell travis right away instead of letting it choose it either way.

Reply:
> Dunno when this started. I just accidentally stumbled on it a couple of days before.

Can you point me to an example? I went looking before I commended earlier but did not find anything. I'm curious which warnings are showing.

> Well since we are testing on Linux anyways we could tell travis right away instead of letting it choose it either way.

It's not going to switch to something else. Or rather, it is not going to break basically all users. Which is what we care about in the end right?

Reply:
> Can you point me to an example? I went looking before I commended earlier but did not find anything. I'm curious which warnings are showing.

See the view config tab: https://travis-ci.org/github/SemanticMediaWiki/SemanticExtraSpecialProperties/builds/664949208/config?utm_medium=notification&utm_source=github_status

> It's not going to switch to something else. Or rather, it is not going to break basically all users. Which is what we care about in the end right?

Still this avoids this nasty notice which drives me crazy.

Reply:
Huh, never saw that. Not sure why you even look there when everything is working fine.

If you do not add the dist key, then there will always be `info` messages. So maybe just deal with the `warning` ones? Which is just the `sudo` key.

Reply:
Not that it matters in a single repo IMO. I'm just saying that I would not add the extra stuff, and am not keen on having it in any of my own repos.

Reply:
I think we should get rid of the warnings and notices where it is sensible. I believe this is the case with all but the distro notice since the latter is the only one which sees changes over time.

Reply:
>  am not keen on having it in any of my own repos.

No worries.